packages/common/src/components/utils/Accordion.tsx

```diff
@@ -73,12 +96,14 @@ interface Props {
 const Accordion: React.FC<Props> = ({
   icon,
   title,
+  titleClose,
   titleId,
   description,
   additionalStyles,
   displayMandatory,
```

Comment:
We can make sure this has a default value of false, so we don't need to assign `displayMandatory as boolean` in places.